python/python/tests/tests_server_modules/test_json.py

```diff
@@ -359,3 +359,68 @@ async def test_json_clear(self, glide_client: TGlideClient):
 
         with pytest.raises(RequestError):
             await json.clear(glide_client, "non_existing_key", ".")
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_json_resp(self, glide_client: TGlideClient):
+        key = get_random_string(5)
+
+        json_value = '{"obj":{"a":1, "b":2}, "arr":[1,2,3], "str": "foo", "bool": true, "int": 42, "float": 3.14, "nullVal": null}'
+        assert await json.set(glide_client, key, "$", json_value) == OK
+
+        assert await json.resp(glide_client, key, "$.*") == [
+            [b"{", [b"a", 1], [b"b", 2]],
```

Comment:
Could you please help me to understand why the first element of this list is a open curly bracket? How could a JSONPath search of `$.*` end up with such result? Or is there anything wrong with our `json_value` test data to begin with?